### ISSUE TITLE
useAnimate question, not a bug

### ISSUE THREAD

**Issue by hthief** [bluebill1049/react-simple-animate#63]
Hi, 

Simple question regarding the useAnimate hook. I was making a slider that is controlled by a prop and before I simply used that prop to revert the animation I was trying something like this:

```javascript

const { play, style } = useAnimate({
        start: { transform: `translateY(${showContent ? '100%' : '0%'})`},
        end: { transform:`translateY(${showContent ? '0': '100%'})` },
    });
useEffect(() => {
        play(true);
    }, [show]);
```
My question is more of a curiosity, but why doesn't the first implementation work like expected? Is it a misunderstanding of how the useEffect hook works?

**Comment by bluebill1049** [comment]
What do u mean first implementation works?

**Comment by hthief** [comment]
@bluebill1049 no, it doesn't but I thought it would, since the prop is changed I figured the animate hook would give a different style object, but it returns the same in any case

**Comment by bluebill1049** [comment]
What’s this showContent doing there?

**Comment by hthief** [comment]
It's a control prop that I use to reverse the animation.
Before I realised I could just do the same thing with the following code, I tried the first approach:
````javascript

const { play, style } = useAnimate({
        start: { transform: 'translateY(100%)' },
        end: { transform: 'translateY(0%)' },
    });

    useEffect(() => {
        play(showContent);
    }, [showContent]);
`````

And was left wondering what could be wrong about the first approach

**Comment by bluebill1049** [comment]
I think this is the right solution which you posted:
```
const { play, style } = useAnimate({
        start: { transform: 'translateY(100%)' },
        end: { transform: 'translateY(0%)' },
    });

    useEffect(() => {
        play(showContent);
    }, [showContent]);
```

i haven't test it, but look at this example, your animation probably would stay the same, because RSA wouldn't be able to change the style because play is not been called
```
const { play, style } = useAnimate({
        start: { transform: `translateY(${showContent ? '100%' : '0%'})`},
        end: { transform:`translateY(${showContent ? '0': '100%'})` },
    });
useEffect(() => {
        play(true);
    }, [show]);
```

**Comment by hthief** [comment]
oh! 😓  i see now what you meant about the previous question about showContent, `show` was the original prop name but I changed when copying it to the comments here.
You know what, I should have linked a sandbox from the start:
https://codesandbox.io/embed/blazing-currying-qf9zz

in the example the first box is using the non-working impl, while the second uses the hooks.

sorry about the confusion 😬
